assets/js/googlesitekit/widgets/util/constants.js

```diff
@@ -28,3 +28,8 @@ export const WIDTH_GRID_COUNTER_MAP = {
 };
 
 export const HIDDEN_CLASS = 'googlesitekit-hidden';
+export const SPECIAL_WIDGET_STATES = [
+	'ReportZero',
+	'ModuleActivateCTA',
+	'CompleteModuleActivationCTA',
```

Comment:
I don't think we should reference component names as strings here. We should be able to import those components directly for comparison where needed. E.g. https://github.com/google/site-kit-wp/blob/1198c7ed8bcaaf09dd7fc9597be17c2352dbfc97/assets/js/googlesitekit/widgets/util/is-inactive-widget-state.js#L33

We preserve the names of component functions in minified production builds, but if that were to ever change I think this name based reference would break.